crates/bevy_pbr/src/light/mod.rs

```diff
@@ -496,12 +496,19 @@ pub enum ShadowFilteringMethod {
     Temporal,
 }
 
+/// System sets used to run light-related systems.
 #[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
 pub enum SimulationLightSystems {
     AddClusters,
     AssignLightsToClusters,
+    /// System order ambiguities between systems in this set are ignored:
+    /// each [`build_directional_light_cascades`] system is independent of the others,
+    /// and should operate on distinct sets of entities.
     UpdateDirectionalLightCascades,
```

Comment:
(out of scope for this PR)

is this an opportunity to add a macro to mark that ambiguity is allowed there ? (for better type safety, avoid outdated comments)